pkgs/by-name/ra/ratt/package.nix

```
@@ -2,24 +2,39 @@
   buildGoModule,
   fetchFromSourcehut,
   lib,
+  scdoc,
+  installShellFiles,
 }:
 buildGoModule rec {
   pname = "ratt";
-  version = "unstable-2023-02-12";
+  version = "unstable-2025-03-10";
```

```suggestion
  version = "0-unstable-2025-03-10";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

---

Thanks! Updated the version :)